proto/kira/gov/network_properties.proto

```diff
@@ -20,35 +20,39 @@ enum NetworkProperty {
     VOTE_QUORUM = 2 [(gogoproto.enumvalue_customname) = "VoteQuorum"];
     PROPOSAL_END_TIME = 3 [(gogoproto.enumvalue_customname) = "ProposalEndTime"];
     PROPOSAL_ENACTMENT_TIME = 4 [(gogoproto.enumvalue_customname) = "ProposalEnactmentTime"];
-    ENABLE_FOREIGN_TX_FEE_PAYMENTS = 5 [(gogoproto.enumvalue_customname) = "EnableForeignFeePayments"];
-    MISCHANCE_RANK_DECREASE_AMOUNT = 6 [(gogoproto.enumvalue_customname) = "MischanceRankDecreaseAmount"];
-    INACTIVE_RANK_DECREASE_PERCENT = 7 [(gogoproto.enumvalue_customname) = "InactiveRankDecreasePercent"];
-    POOR_NETWORK_MAX_BANK_SEND = 8 [(gogoproto.enumvalue_customname) = "PoorNetworkMaxBankSend"];
-    MIN_VALIDATORS = 9 [(gogoproto.enumvalue_customname) = "MinValidators"];
-    JAIL_MAX_TIME = 10 [(gogoproto.enumvalue_customname) = "JailMaxTime"];
-    ENABLE_TOKEN_WHITELIST = 11 [(gogoproto.enumvalue_customname) = "EnableTokenWhitelist"];
-    ENABLE_TOKEN_BLACKLIST = 12 [(gogoproto.enumvalue_customname) = "EnableTokenBlacklist"];
+    MIN_PROPOSAL_END_BLOCKS = 5 [(gogoproto.enumvalue_customname) = "MinProposalEndBlocks"];
```

Comment:
We need to start to keep the numbers as they were, in protocol buffers in order to be compatible with changes, the number after the equal should not be changed. Now maybe is not a problem, but this in production means the chain is broken.

Reply:
yes, for now, we need to make things organized with numbers before going production.